dffml/operation/io.py

```diff
@@ -25,13 +25,14 @@
 class AcceptUserInputContext(OperationImplementationContext):
     @staticmethod
     def receive_input():
+        print("Enter the value:")
```

Comment:
```suggestion
        print("Enter the value: ", end="")
```

This way the user can type in input on the same line